### DISCUSSION TITLE
Student Package

### DISCUSSION THREAD

**Discussion by marwaynss** [community/community#87388] [GitHub Education]
### Select Topic Area

General

### Body

Hello

**Comment by marwaynss** [comment]
I applied to student package and still I didn’t get any response

**Reply by medu548** [reply]
I am having the same issue! I was approved but have not received any further information

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @marwaynss , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
